config/index.js

```diff
@@ -63,6 +63,8 @@ f.sensorRefresh = process.env['MATRIX_SENSOR_REFRESH'] || 2500;
 // not in yet
 f.sensorSockets = true;
 f.socketCheckDelay = 60000;
+f.registrationUUID = "b1a6752152eb4d36e13e357d7c225466";
```

Comment:
Nope. On a side note, it might probably change in the future if we register one with Bluetooth SIG